README.Rmd

```diff
@@ -316,35 +319,50 @@ param_notes <- param_tbl$value %>%
   )) %>%
   unlist()
 
-ccao::vars_dict %>%
-  inner_join(
-    param_tbl %>% mutate(description = param_notes),
-    by = c("var_name_model" = "value")
+param_tbl_fmt <- param_tbl %>%
+  mutate(description = param_notes) %>%
+  left_join(
+    ccao::vars_dict,
+    by = c("value" = "var_name_model")
```

Comment:
I switched this up so that the params are the left side of the join here, which feels more intuitive to me than `vars_dict` being the left side. We also use a left join so that we'll preserve all of the parameters, even if one happens to be misdocumented in `vars_dict` in the future.